webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -0,0 +1,93 @@
+.ldap-wizard-wrapper {
+    background-color: rgba(var(--center-channel-color-rgb), 0.04);
+
+    .ldap-wizard-content-wrapper {
+        display: flex;
+        overflow: hidden; // Prevent double scrollbars on the body if content overflows
+        flex-grow: 1;
+        padding-top: 20px;
+    }
+
+    
+    .ldap-wizard-sidebar {
+        position: sticky;
+        top: 0;
+        display: flex;
+        width: 319px;
```

Comment:
Could you change this to not be a fixed width? It seems like this won't scale well when translated text is a different length

Reply:
Yep, I removed it -- now I think the width is mostly contrained by the padding the overall flex container. 
Some Qs for @asaadmahmood:
- can you double check `.ldap-wizard-sidebar`'s padding and maybe add a min-width if you want?